src/openforms/appointments/contrib/jcc/plugin.py

```diff
@@ -276,6 +276,11 @@ def create_appointment(
             FIELD_TO_XML_NAME[key]: value for key, value in client.details.items()
         }
 
+        # ensure start_at is naive, as JCC does not handle ISO-8601 datetimes with TZ
+        # information.
+        if not timezone.is_naive(start_at):
+            start_at = timezone.make_naive(start_at, timezone=TIMEZONE_AMS)
```

Comment:
Shouldn't this be set to `settings.TIME_ZONE`?

Reply:
that is the (implicit) default in Django, but we've had projects where we configured this django setting to be UTC which would yield the wrong results in this case.

JCC itself communicates all datetimes in the Dutch timezone, which is `Europe/Amsterdam`, so our code interfacing with JCC must be pinned to that value. settings.TIME_ZONE also being `Europe/Amsterdam` is coincidence.